app/views/works/index.html.erb

```diff
@@ -0,0 +1,87 @@
+<h2>List of Works</h2>
+
+<%# table here %>
+<section class="media-table">
+    <h4>Albums</h4>
```

Comment:
Note that you repeat these sections a lot for each category... Maybe use a partial for this?